src/main/java/jarvis/logic/parser/AddNotesCommandParser.java

```diff
@@ -31,7 +31,7 @@ public AddNotesCommand parse(String args) throws ParseException {
 
         Index lessonIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_LESSON_INDEX).get());
         String studentPrefix = argMultimap.getValue(PREFIX_STUDENT_INDEX).orElse("-1");
-        Index studentIndex = studentPrefix == "-1" ? null : ParserUtil.parseIndex(studentPrefix);
+        Index studentIndex = studentPrefix.equals("-1") ? null : ParserUtil.parseIndex(studentPrefix);
```

Comment:
Now that I think about it this might create issues when the user actually passes in "-1" for the student index. This will not throw a ParseError.

 I think a better way to implement this would be to use `argMultimap.getValue(prefix).isPresent()` (like in the function `arePrefixesPresent` a few lines below) and an if-else statement to check if the user has actually passed in a student index, and then only parse the userstring if the user entered in the prefix.